src/Module.zig

```diff
@@ -1111,6 +1111,10 @@ pub const Scope = struct {
             return buf.toOwnedSliceSentinel(0);
         }
 
+        pub fn fullPath(file: File, gpa: *Allocator) ![]u8 {
```

Comment:
Nit: you name the param gpa, but in all the use sites, you pass an arena. This could be a little confusing for some people.

Reply:
Fixed.